experimental/telemetry/doc.go

```diff
@@ -32,22 +32,33 @@
 //	    tracker         telemetry.Tracker
 //	}
 //
-// 2. Start the telemetry client on plugin activate
+// 2. Start the telemetry client and tracker on plugin activate
 //
 //	func (p *Plugin) OnActivate() error {
-//	    p.telemetryClient, err = telemetry.NewRudderClient()
-//	    if err != nil {
-//	        p.API.LogWarn("telemetry client not started", "error", err.Error())
-//	    }
+//		p.telemetryClient, err = telemetry.NewRudderClient()
+//		if err != nil {
+//			p.API.LogWarn("telemetry client not started", "error", err.Error())
+//		}
+//		...
+//		p.tracker = telemetry.NewTracker(
+//			p.telemetryClient,
+//			p.API.GetDiagnosticId(),
+//			p.API.GetServerVersion(),
+//			Manifest.Id,
+//			Manifest.Version,
+//			"plugin_short_namame",
+//			telemetry.NewTrackerConfig(p.API.GetConfig()),
+//			logger.New(p.API)
+//		)
 //	}
```

Comment:
Actually, the indenting you are using there is correct, and the existing one is wrong. Here tabs are used correctly, instead of spaces. Would you be interested in updating also the other code example here?